woocommerce/wc-admin
# Update returning customer values to respect excluded statuses

Fixes #1473 

Doesn't count orders that fall within excluded statuses as first-time orders and retroactively updates orders when a status change causes a shift in the original first order.

### Detailed test instructions:

1.  Create an order with a customer (don't use a guest until https://github.com/woocommerce/wc-admin/pull/1479 is merged).
2.  Make sure that order status is set to an excluded status.
3.  Make another order with that same customer and set to a non-excluded status.
4.  Check that the 2nd order is marked as a new customer.
5.  Update the 1st order to a non-excluded status.
6.  Check that the first order (assuming it has an earlier creation date) is now set to "New" and the 2nd order is set to "Returning."
7.   Make sure all tests pass.
